routing.go

```diff
@@ -13,11 +13,11 @@ import (
 	pstore "github.com/libp2p/go-libp2p-peerstore"
 )
 
-// ErrNotFound is returned when a search fails to find anything
+// ErrNotFound is returned when a get fails to find anything.
 var ErrNotFound = errors.New("routing: not found")
 
-// ErrNotSupported is returned when a search or put fails because the key type
-// isn't supported.
+// ErrNotSupported is returned when a put fails because the key type isn't
+// supported.
```

Comment:
It's used for a bit more than that, for example - https://github.com/libp2p/go-libp2p-routing-helpers/blob/master/composed.go#L52